third_party/git-cmake-format/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 message(STATUS "Configuring and building git-cmake-format")
 ginkgo_load_git_package(git-cmake-format
     "https://github.com/ginkgo-project/git-cmake-format.git"
-    "29c23665d624e1cae1308bec651706fdaa8fe38b"
-    "-DGCF_CLANGFORMAT_MINIMAL_VERSION=5.0.0")
```

Comment:
Should we change it from 5 to 8?
5 works most of ginkgo but some corner cases are different from 8.
If we decide to require 8+, also need to update the ReadME

Reply:
Good point! Since clang-format only matters for `GINKGO_DEVEL_TOOLS=ON`, I would still require 8 though, otherwise code needs to be reformatted every time somebody creates a PR with one of those files changed.

Reply:
make sense. thanks